### ISSUE TITLE
Add focus styles for relation fields

### ISSUE THREAD

**Issue by jmauzyk** [craftcms/cms#6002]
### Description

Currently relations fields have no focus styles so when using tab to navigate through entry fields it is unclear when something like *Add an asset* is focused. Would be helpful if relations fields could have a light gray background when focused, similar to the gray background for focused relations.

### Additional info

- Craft version: Pro 3.4.17.1
- PHP version: 7.4.5
- Database driver & version: MySQL 5.5.5

**Comment by brandonkelly** [comment]
Just improved relational fields’ focus styling for the next 3.5 release.

**Comment by brandonkelly** [comment]
Craft 3.5 RC6 is out now with this change.
